app/src/main/res/layout/fragment_pdf_to_image.xml

```diff
@@ -58,37 +58,60 @@
             android:layout_height="wrap_content"
             android:layout_alignParentBottom="true"
             android:layout_centerHorizontal="true"
-            android:gravity="center"
+            android:orientation="vertical"
             android:visibility="gone">
 
             <Button
-                android:id="@+id/shareImages"
+                android:id="@+id/viewImagesinGallery"
```

Comment:
`i` should be capital.

Reply:
@jddeep Fix these formatting changes too